fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -167,6 +167,10 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :log_path,
                                        env_name: "FL_CARTHAGE_LOG_PATH",
                                        description: "Path to the xcode build output",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :executable,
+                                       env_name: "FL_CARTHAGE_EXECUTABLE",
+                                       description: "Path to the `carthage` executable on your machine",
                                        optional: true)
```

Comment:
Instead of making this optional can you try making this have `default_value: "carthage"`? This should also clean up the code on line 8 🙃

Reply:
Addressed your feedback @joshdholtz.